source/common/stats/stats_impl.h

```diff
@@ -19,6 +19,7 @@
 #include "common/common/assert.h"
 #include "common/common/hash.h"
 #include "common/common/non_copyable.h"
+#include "common/common/thread.h"
```

Comment:
IWYU? :)

Reply:
I can switch this to make a direct dependence to absl/synchronization.h.  WDYT though of leaving thread.h as an intermediary?  I think the bazel deps feature is a little easier to work with than the external_deps feature.

Reply:
Probably not something I'd bike shed about if this is deliberate, I'm not very consistent here in the Envoy code base.